packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/helpers/isFollowedByParticipleSpec.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line max-len
+import { isParticiple, isFollowedByParticiple, isNotFollowedByParticiple } from "../../../../../src/scoring/assessments/inclusiveLanguage/helpers/isFollowedByParticiple";
+
+describe( "test the isParticiple function", () => {
+	it( "returns true if a word is a regular participle", () => {
+		expect( isParticiple( "worked" ) ).toBeTruthy();
+	} );
+	it( "returns true if a word is an irregular participle", () => {
+		expect( isParticiple( "awoken" ) ).toBeTruthy();
+	} );
+	it( "returns false if a word is not a participle", () => {
+		expect( isParticiple( "cat" ) ).toBeFalsy();
+	} );
+} );
+
+describe( "test isFollowedByParticiple and isNotFollowedByParticiple", () => {
```

Comment:
Same would apply here (splitting it up into two `describe` calls), but `isNotFollowedByParticiple` will most likely be removed 😄 .

Reply:
Indeed. :-)